### DISCUSSION TITLE
Generate JMeter performance testing scripts.

### DISCUSSION THREAD

**Discussion by shalousun** [TongchengOpenSource/smart-doc#525] [Ideas]
# Problem Solving
Use the documentation data generated by the current smart-doc scan to generate JMeter performance testing scripts. Automating the generation of performance testing scripts can significantly reduce the time testers spend writing them and improve team efficiency.
# Development Steps
- You can write a Java Spring Boot project, find colleagues who are skilled in performance testing to compile a performance testing script, and then abstract and transform it into a template on the smart-doc side.
- Develop functionality to generate performance testing scripts based on the transformed template, and import the automatically generated performance testing scripts into JMeter for validation.
# Developers
- All developers interested in contributing are welcome to comment on this proposal.
- Interested individuals can also join the official smart-doc technical group or search for @shalousun on WeChat to contact the author of smart-doc and participate in development.

**Comment by chenqi146** [comment]
#668
